### ISSUE TITLE
[Bug Report][3.7.0] VCombobox and VAutocomplete - keyboard navigation over the items does not work properly

### ISSUE THREAD

**Issue by b-maslennikov** [vuetifyjs/vuetify#20850]
### Environment
**Vuetify Version:** 3.7.0
**Last working version:** 3.6.15
**Vue Version:** 3.5.13
**Browsers:** Google Chrome, Mozilla Firefox, Microsoft Edge
**OS:** Windows, Linux

### Steps to reproduce
1) add `v-combobox` or `v-autocomplete` with `multiple` and `:items`
2) try to select all items one by one using keyboard keys (`↓` (down arrow) to navigate and `␣` (space) to select)

### Expected Behavior
after selecting the element should be enough to press down arrow only once to navigate to the next item
example: you need to press:  `␣ ↓ ␣ ↓ ␣` to select first 3 items in the list

### Actual Behavior
focused item "resets" to the first item after current item selecttion
you need to press `␣ ↓ ↓ ␣ ↓ ↓ ↓ ␣` to select first 3 items in the list

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNptj8sKwjAQRX8lzCab2gguhBIFv6PtIomjBvKiTYog/rtj7aKKiwMzF85wp33AOBhxSqmeCkIDMqNPTmU8doExOW1USvM4LyaGrGzAYYmW0Ouo45354rJNDllj6cp46KDlildcE4Y4874DJtauKjmST1LGvz6SdyGuxO3Ll+K3zzv51JVi9QY8q1Ccq2BX7+st9C/Cb0us)

### Other comments
Probably it's been broken by [b9ed79d](https://github.com/vuetifyjs/vuetify/commit/b9ed79db8ae68eed6bc13cf10aefa6944fa8675b)
